.github/workflows/build.yml

```diff
@@ -280,9 +346,9 @@ jobs:
         with:
           bundler-cache: true
 
-      - name: Install Fastlane, Mint, xcbeautify, and yq
+      - name: Install Fastlane, Mint
         run: |
-          brew install fastlane mint xcbeautify yq
+          brew install fastlane mint
```

Comment:
🤔 Has `xcbeautify` been replaced by something else?

Reply:
It's part of the image already so it doesn't need to be installed; it was throwing warnings about this (and yq) on previous builds.